servicemesh-install/topics/install-overview.adoc

```diff
@@ -23,11 +23,13 @@ The operator creates an Ansible job that runs an Ansible playbook that performs
 ** istio-telemetry
 * Installs Elasticsearch
 * Installs Grafana
-* Installs the following Jaeger components (if configured in the custom resource definition):
+* Installs the following Jaeger components:
 ** jaeger-agent
 ** jaeger-collector
 ** jaeger-query
 * Installs Prometheus
+* Installs the following Kiali components (if configured in the custom resource definition):
+** kiali
 * Performs the following launcher configuration tasks (if configured in the custom resource definition):
 ** Creates a `devex` project and installs the Fabric8 launcher into that project.
-** Adds the cluster admin role to the OpenShift user specified in the launcher parameters in the custom resource file.
\ No newline at end of file
+** Adds the cluster admin role to the OpenShift user specified in the launcher parameters in the custom resource file.
```

Comment:
s/OpenShift/{product-title} (multiple places, multiple files)

Reply:
This should now be done